web/src/components/ak-wizard-main/AkWizard.ts

```diff
@@ -50,27 +57,38 @@ export class AkWizard<D, Step extends WizardStep = WizardStep>
 
     description?: string;
 
-    wizard: AkWizardController<D>;
-
     constructor(prompt: string, header: string, description?: string) {
         super();
         this.header = header;
         this.prompt = prompt;
         this.description = description;
-        this.wizard = new AkWizardController(this);
+    }
+
+    connectedCallback() {
```

Comment:
You can quickly see just how little of the controller was left to be moved after replacing all the "find next/prev by index" safety stuff was removed in favor of a constrained state machine.